library/src/test/java/com/okta/appauth/android/ConfigurationStreams.java

```diff
@@ -27,42 +27,91 @@
             "{" +
-            "  \"client_id\": \"example_client_id\"," +
-            "  \"redirect_uri\": \"com.okta.appauth.android.test:/oauth2redirect\"," +
-            "  \"scopes\": [" +
-            "    \"openid\"," +
-            "    \"foo\"" +
-            "  ]," +
-            "  \"issuer_uri\": \"https://example.com/issuer\"" +
-            "}";
+                    "  \"client_id\": \"example_client_id\"," +
+                    "  \"redirect_uri\": \"com.okta.appauth.android.test:/oauth2redirect\"," +
+                    "  \"end_session_redirect_uri\": \"com.okta.appauth.android.test:/logout\"," +
+                    "  \"scopes\": [" +
+                    "    \"openid\"," +
+                    "    \"foo\"" +
+                    "  ]," +
+                    "  \"issuer_uri\": \"https://example.com/issuer\"" +
+                    "}";
 
     public static InputStream getExampleConfiguration() {
         return new ByteArrayInputStream(EXAMPLE_JSON_CONFIG.getBytes());
     }
 
     public static final String OTHER_JSON_CONFIG =
             "{" +
-            "  \"client_id\": \"other_client_id\"," +
-            "  \"redirect_uri\": \"com.okta.appauth.android.test:/oauth2redirect\"," +
-            "  \"scopes\": [" +
-            "    \"openid\"," +
-            "    \"bar\"" +
-            "  ]," +
-            "  \"issuer_uri\": \"https://example.com/other-issuer\"" +
-            "}";
+                    "  \"client_id\": \"other_client_id\"," +
+                    "  \"redirect_uri\": \"com.okta.appauth.android.test:/oauth2redirect\"," +
+                    "  \"end_session_redirect_uri\": \"com.okta.appauth.android.test:/logout\"," +
+                    "  \"scopes\": [" +
+                    "    \"openid\"," +
+                    "    \"bar\"" +
+                    "  ]," +
+                    "  \"issuer_uri\": \"https://example.com/other-issuer\"" +
+                    "}";
 
     public static InputStream getOtherConfiguration() {
         return new ByteArrayInputStream(OTHER_JSON_CONFIG.getBytes());
     }
 
     public static final String INVALID_JSON_CONFIG =
             "{" +
-            "  \"client_id\": \"example_client_id\"," +
-            "  \"redirect_uri\": \"com.okta.appauth.android.test:/oauth2redirect\"," +
-            "  \"scopes\": [" +
-            "  ]," +
-            "  \"issuer_uri\": \"https://example.com/issuer\"" +
-            "}";
+                    "  \"client_id\": \"example_client_id\"," +
+                    "  \"redirect_uri\": \"com.okta.appauth.android.test:/oauth2redirect\"," +
+                    "  \"end_session_redirect_uri\": \"com.okta.appauth.android.test:/logout\"," +
+                    "  \"scopes\": [" +
+                    "  ]," +
+                    "  \"issuer_uri\": \"https://example.com/issuer\"" +
+                    "}";
+
+    public static final String VALID_AUTHORIZATION_RESPONSE =
```

Comment:
Feel free to move these strings to json files if it is easier for you to manage (your call)

Reply:
Let's keep it as it is for now.